networkit/graph.pyx

```diff
@@ -665,6 +665,186 @@ cdef class Graph:
 			yield dereference(it)
 			preincrement(it)
 
+	def attachNodeAttribute(self, name, ofType):
+		"""
+		Attaches a node attribute to the graph and returns it
+
+		A = G.attachNodeAttribute("attributeIdentifier", ofType)
+
+		all values are initially undefined
+
+		for existing nodes values can be set by
+
+		A[node] = value
+
+		and get by
+
+		value = A[node]
+
+		getting undefined values raises a ValueError
+		removing a node makes all its attributes undefined
+
+		Parameters:
+		-----------
+		name   : a distinguished name for this attribute
```

Comment:
Format for parameters is:
```
name : str
	A distinguished name for this attribute.
ofType : int or float or str
	The type of this attribute.
```
(Again, this is needed in order to preserve correct rendering of doc)